hw03_oludairoa.py

```diff
@@ -0,0 +1,108 @@
+#################################################################################
+# Author: Ayomide Oludairo
+# Username: oludairoa
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Learn how to use github push and pull requests
+# Google Doc Link: https://docs.google.com/document/d/1yX8V99iRkV56ZTmNQmI9XwANLQ7LLj5dzj0nO0i1hgw/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+def draw_rect(t, sz, sz2):
+    """
+    The draw square function creates the 4 sides of a rectangle (can be a square).
+    Takes 3 inputs, the turtle t and the length of sides with 2 being parallel to each other on each side sz and sz2.
+    """
+
+    if int(sz) > 200:
+        return "That value is too large try something smaller"
+
+    # The code inside the for loop creates one right angle of a rectangle, so the loop is used to do it twice.
+    for i in range(2):
+        t.forward(sz)
+        t.left(90)
+        t.forward(sz2)
+        t.left(90)
+
+    # Puts the turtle back in its natural eastward facing position
+    t.right(90)
+
+def draw_parallelogram(t, sz, sz2):
+    """
+    Creates a parallelogram that will be the sides of the walls.
+    This function must be called after the rectangle function as it begins where the rectangle ends.
+    """
+    t.left(45)
+    t.forward(sz)
+    t.left(45)
+    t.forward(sz2)
+    t.left(135)
+    t.forward(sz)
+
+def draw_roof(t, sz):
+    """"
+    This function will create a triangle that will serve as a roof for this shed
+    Must be called after the parallelogram as it begins where the parallelogram ends.
+    """
+    t.right(90)
+    t.forward(sz)
+    t.left(90)
+    t.forward(sz)
+
+# Defining the main function where the code will run
+def main():
+    """
+    Initialize two separate turtles that will create the body of the shed and a door.
+    """
+    wn = turtle.Screen()
+    wn.colormode(225) # Enables the use of RGB color identification mode
+    wn.bgcolor((173/255, 216/255, 230/255)) # Colors in rgb mode are represented in bits (0-1)
```

Comment:
When you change the colormode to 255 (line above this line), your values should then be between 0 and 255. It looks like your number are being divided by 255, making them 1/255th as bright as you expect them to be.

Reply:
`wn.bgcolor(173, 216, 230)`